editions/tw5.com/tiddlers/filters/examples/addprefix.tid

```diff
@@ -1,7 +1,17 @@
 created: 20150118132851000
-modified: 20150118183116000
+modified: 20250323034523441
 tags: [[addprefix Operator]] [[Operator Examples]]
 title: addprefix Operator (Examples)
 type: text/vnd.tiddlywiki
 
 <<.operator-example 1 "Cat Garden [[Favourite Armchair]] +[addprefix[My ]]">>
+
+```
```

Comment:
These instructions are formatted as monospaced text. We should use ordinary wikitext for documentation.